src/Compilers/CSharp/Test/Semantic/Semantics/ReadOnlyStructsTests.cs

```diff
@@ -289,6 +304,93 @@ public struct S
             comp.VerifyDiagnostics();
         }
 
+        [Fact]
+        public void ReadOnlyMethod_CallBaseMethod()
```

Comment:
>ReadOnlyMethod_CallBaseMethod [](start = 20, length = 29)

Minor point: perhaps split into 4 separate tests so it's clear which warnings apply to each.

Reply:
Done